copter/source/docs/terrain-following-manual-modes.rst

```diff
@@ -4,7 +4,7 @@
 Surface Tracking
 ================
 
-Copter supports low altitude Surface Tracking in nearly all modes including pilot controlled modes like :ref:`AltHold <altholdmode>`, :ref:`Loiter <loiter-mode>` and :ref:`PosHold <poshold-mode>` and also fully autonomous modes including :ref:`AUTO <auto-mode>` and :ref:`Guided <ac2_guidedmode>`.
+Copter supports low altitude Surface Tracking, either a floor/ground or a ceiling, in nearly all modes including pilot controlled modes like :ref:`AltHold <altholdmode>`, :ref:`Loiter <loiter-mode>` and :ref:`PosHold <poshold-mode>` and also fully autonomous modes including :ref:`AUTO <auto-mode>` and :ref:`Guided <ac2_guidedmode>`.
 
 Additionally, Terrain Following is supported in autonomous modes using a terrain height data base stored on the autopilot's SD card and/or obtained real-time from the Ground Control Station.
```

Comment:
We should change "Terrain Following" to be a hyperlink to the terrain following page.  This description that Terrain Following only uses the terrain database is also incorrect because a lidar can also be used.